lego-mindstorms-pieces.py

```diff
@@ -340,9 +374,16 @@ def order(shop=None, browser=None, lego_set=None, order_list=None, username=None
                 print()
 
         except NoSuchElementException:
-            print("OOOPS! No LEGO part with that number found in set #{set}. :-(".format(
-                set=lego_set))
-            not_in_set_counter += 1
+
+            if is_electric_part(part_no):
+                print("For Lego, Electric part are not in set #{set}, see note at the end.".format(
```

Comment:
I don't fully understand the sentence. Do you intend to say, The LEGO Group treats electric parts separately, as not being part of the set?

Can you re-phrase this a bit? I would guess the sentence is currently French ("Pour LEGO, c'est ..."), n'est-ce pas?

Please also write "LEGO" all-caps (as on the company website) and "electric" all small caps. Merci !